<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ManyToManyWidget</class>
 <widget class="QWidget" name="ManyToManyWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>702</width>
    <height>468</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout">
     <property name="spacing">
      <number>1</number>
     </property>
     <property name="margin">
      <number>0</number>
     </property>
     <item>
      <widget class="QLineEdit" name="uiText"/>
     </item>
     <item>
      <widget class="QToolButton" name="pushAdd">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="focusPolicy">
        <enum>Qt::NoFocus</enum>
       </property>
       <property name="text">
        <string>&amp;Add</string>
       </property>
       <property name="icon">
        <iconset resource="common.qrc">
         <normaloff>:/images/new.png</normaloff>:/images/new.png</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="pushRemove">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="focusPolicy">
        <enum>Qt::NoFocus</enum>
       </property>
       <property name="text">
        <string>&amp;Remove</string>
       </property>
       <property name="icon">
        <iconset resource="common.qrc">
         <normaloff>:/images/delete.png</normaloff>:/images/delete.png</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="Line" name="line">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="pushBack">
       <property name="text">
        <string>...</string>
       </property>
       <property name="icon">
        <iconset resource="common.qrc">
         <normaloff>:/images/previous.png</normaloff>:/images/previous.png</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="uiLabel">
       <property name="text">
        <string>(0/0)</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="pushForward">
       <property name="text">
        <string>...</string>
       </property>
       <property name="icon">
        <iconset resource="common.qrc">
         <normaloff>:/images/next.png</normaloff>:/images/next.png</iconset>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="Screen" name="screen" native="true">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>Screen</class>
   <extends>QWidget</extends>
   <header>Koo.Screen.Screen</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="common.qrc"/>
 </resources>
 <connections/>
</ui>
